docs/search.md

```diff
@@ -37,6 +37,63 @@ curl -XGET 'https://…/api/v1/search' -d '{
 }'
 ```
 
+## Advanced Search 
+
+Advanced Search is based on [Elasticsearch](https://www.elastic.co/). It comprises of spatio-temporal search and keyword search.
+
+### Example requests
+
+- [http://o2r.uni-muenster.de/api/v1/search?q=*&from&to&coords&keyword](http://o2r.uni-muenster.de/api/v1/search?q=*&from&to&coords&keyword)
+- [http://o2r.uni-muenster.de/api/v1/search?q=*&from=Wed+Apr+02+2014+00%3A00%3A00+GMT%2B0200+%28CEST%29&to=Fri+Jun+09+2017+00%3A00%3A00+GMT%2B0200+%28CEST%29%0D%0A&coords=%5B%5B%5B-74.33349609375%2C55.44771083630114%5D%2C%5B-74.33349609375%2C83.91787587317228%5D%2C%5B7.22900390625%2C83.91787587317228%5D%2C%5B7.22900390625%2C55.44771083630114%5D%2C%5B-74.33349609375%2C55.44771083630114%5D%5D%5D&keyword=geosciences](http://o2r.uni-muenster.de/api/v1/search?q=*&from=Wed+Apr+02+2014+00%3A00%3A00+GMT%2B0200+%28CEST%29&to=Fri+Jun+09+2017+00%3A00%3A00+GMT%2B0200+%28CEST%29%0D%0A&coords=%5B%5B%5B-74.33349609375%2C55.44771083630114%5D%2C%5B-74.33349609375%2C83.91787587317228%5D%2C%5B7.22900390625%2C83.91787587317228%5D%2C%5B7.22900390625%2C55.44771083630114%5D%2C%5B-74.33349609375%2C55.44771083630114%5D%5D%5D&keyword=geosciences)
+
+### Advanced Search Query
```

Comment:
What is this used for? This is the internal query, right? This is the "template" that the above information is filled into and then used for the actual query to elasticsearch?

_Imagine you look at this for the first time ever as a developer!_ Would you really understand what this all means?